src/bin/cargo/commands/info.rs

```diff
@@ -0,0 +1,39 @@
+use cargo::ops::info;
+use cargo::util::command_prelude::*;
+use cargo_util_schemas::core::PackageIdSpec;
+
+pub fn cli() -> Command {
+    Command::new("info")
```

Comment:
Where are the integration tests for this command?

Reply:
I will add it later. I just to make sure the main change is on the right track first.

Reply:
We should also call out in the PR if the docs and completions will be merged in this PR or immediately after (with enough time before a beta branch that they don't get split).

I split them for `cargo add` mostly because the CLI was under such heavy churn during the review.  That is less likely to happen here so it seems reasonable to have docs and completions included.